infrastructure/main.tf

```diff
@@ -67,12 +71,41 @@ module "feature-toggle-api" {
     TEST_ADMIN_PASSWORD         = "${local.test_admin_password}"
     TEST_EDITOR_USERNAME        = "${local.test_editor_user}"
     TEST_EDITOR_PASSWORD        = "${local.test_editor_password}"
+    CMC_ADMIN_USERNAME          = "${data.azurerm_key_vault_secret.cmc_admin_username.value}"
+    CMC_ADMIN_PASSWORD          = "${data.azurerm_key_vault_secret.cmc_admin_password.value}"
+    CMC_EDITOR_USERNAME         = "${data.azurerm_key_vault_secret.cmc_editor_username.value}"
+    CMC_EDITOR_PASSWORD         = "${data.azurerm_key_vault_secret.cmc_editor_password.value}"
     // silence the "bad implementation" logs
     LOGBACK_REQUIRE_ALERT_LEVEL = false
     LOGBACK_REQUIRE_ERROR_CODE  = false
   }
 }
 
+data "azurerm_key_vault" "cmc_key_vault" {
```

Comment:
Is it worth moving cmc related vault stuff like `cmc_key_vault` ,`cmc_admin_username` etc to a different tf file for e.g `cmcUsers.tf`?

Reply:
done